app/src/utils/DateTimeUtils.js

```diff
@@ -37,6 +37,16 @@ export const dateStringFromTimestamp = (timestamp) => {
   return isNaN(day) ? null : fullDate;
 };
 
+/** Returns date in DD_MM_YYYY format */
+export const getTodaysDateInFormat = () => {
+  const date = new Date();
+  const day = String(date.getDate()).padStart(2, "0");
+  const month = String(date.getMonth() + 1).padStart(2, "0");
+  const year = date.getFullYear();
+
+  return `${day}_${month}_${year}`;
+};
+
```

Comment:
Instead of creating a new util, Can make use of `epochToDateAndTimeString`, pass it `new Date()` as arg and extract the date from the returned value. wdyt?
[reference](https://github.com/requestly/requestly/pull/1002/files#diff-78dd56e2bc55dd346199b77ba9fe18de59585ba424c50ef40eb76dfd7ea112dcR187)